File: app/src/main/java/me/devsaki/hentoid/json/core/JsonLangSettings.java
package me.devsaki.hentoid.json.core;

import java.util.List;

public class JsonLangSettings {

    public List<JsonLanguage> languages;

    public static class JsonLanguage {
        public String lang_code = null;
        public String flag_country_code = null;
        public String local_name = null;
        public String english_name = null;
    }
}

File: app/src/main/java/me/devsaki/hentoid/json/core/JsonLangSettings.kt
package me.devsaki.hentoid.json.core

import com.squareup.moshi.Json

data class JsonLangSettings(
    val languages: List<JsonLanguage>
) {
    data class JsonLanguage(
        @Json(name = "lang_code")
        val langCode: String,
        @Json(name = "flag_country_code")
        val flagCountryCode: String,
        @Json(name = "local_name")
        val localName: String,
        @Json(name = "english_name")
        val englishName: String
    )
}
